### DISCUSSION TITLE
[Question] Is the final exam scheduled at June 10th 9am-11:45 am?

### DISCUSSION THREAD

**Discussion by tncks0121** [prosyslab-classroom/cs524-program-analysis#304] [Q&A] [answered]
Name: Suchan Park

Is the final exam scheduled at June 10th 9am-11:45 am? (found the information from https://cais.kaist.ac.kr)

Since the schedule does not seem to be posted in this repository, I wanted to be sure.

**Comment by goodtaeeun** [comment, marked as answer]
Yes, the final exam is planned scheduled at June 10th 9am-11:45 am
